raiden/waiting.py

```diff
@@ -95,6 +103,7 @@ def wait_for_participant_newbalance(
             token_address,
             partner_address,
         )
+        log.debug('channel_state', channel_state=channel_state, balance=balance(channel_state))
```

Comment:
remove or add more info (addresses) to make debugging easier

Reply:
This is will go away